.github/workflows/build.yml

```diff
@@ -119,6 +113,15 @@ jobs:
           echo "{\"project\": \"$project\", \"version\": \"$version\", \"id\": $GITHUB_RUN_NUMBER, \"commit\": \"$GITHUB_SHA\"}" > metadata.json
           rsync -P -e "ssh -o StrictHostKeyChecking=no -i id_ecdsa" metadata.json $DOWNLOADS_USERNAME@$DOWNLOADS_SERVER_IP:~/uploads/$project/$GITHUB_RUN_NUMBER/
 
+      - name: Publish to Modrinth
+        uses: gradle/gradle-build-action@v2
+        if: ${{ github.ref_name == 'master' && job.status == 'success' && github.repository == 'GeyserMC/Geyser' }}
```

Comment:
You can replace `job.status == 'success'` with `success()`

Reply:
https://github.com/GeyserMC/Geyser/pull/3644/commits/d0a0d7a583b85d92164be8a8ac01425bb363616b